### PR TITLE
Release notes for Istio 1.4.10

### PR DESCRIPTION
**Pull request by jacob-delgado**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by jacob-delgado** [comment]
Waiting for confirmation on shipping date. The content can be approved.

File: content/en/news/releases/1.4.x/announcing-1.4.10/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Announcing Istio 1.4.10
+linktitle: 1.4.10
+subtitle: Patch Release
+description: Istio 1.4.10 security release.
+publishdate: 2020-06-18
+release: 1.4.10
+aliases:
+    - /news/announcing-1.4.10
+test: n/a
```
Comment (ericvn) [inline content/en/news/releases/1.4.x/announcing-1.4.10/index.md:10]: I don't think the test: tag is needed under en/news. Probably doesn't hurt. @frankbu Will we ever have tests outside of docs?

File: content/en/news/releases/1.4.x/announcing-1.4.10/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Security update
+
+- **ISTIO-SECURITY-2020-006** Excessive CPU usage when processing HTTP/2 SETTINGS frames with too many parameters, potentially leading to a denial of service.
+
+__[CVE-2020-11080](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11080)__: By sending a specially crafted packet, an attacker could cause the CPU to spike at 100%. This could be sent to the ingress gateway or a sidecar.
+
+## Bug fixes
+
+- **Fixed** `istio-cni-node` crash when `COS_CONTAINERD` and Istio CNI are enabled when running on Google Kubernetes Engine ([Issue 23643](https://github.com/istio/istio/issues/23643]))
```
Comment (ericvn) [inline content/en/news/releases/1.4.x/announcing-1.4.10/index.md:30]:
```suggestion
- **Fixed** `istio-cni-node` crash when `COS_CONTAINERD` and Istio CNI are enabled when running on Google Kubernetes Engine ([Issue 23643](https://github.com/istio/istio/issues/23643))
```

File: content/en/news/releases/1.4.x/announcing-1.4.10/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- **Fixed** Istio CNI causing pod initialization to experience a 30-40 second delay on startup when DNS unreachable ([Issue 23770](https://github.com/istio/istio/issues/23770))
```
Comment (ericvn) [inline content/en/news/releases/1.4.x/announcing-1.4.10/index.md:31]:
```suggestion
- **Fixed** Istio CNI causes pod initialization to experience a 30-40 second delay on startup when DNS is unreachable ([Issue 23770](https://github.com/istio/istio/issues/23770))
```

File: content/en/news/releases/1.4.x/announcing-1.4.10/index.md (hunk 1 above)
Comment (jacob-delgado) [reply content/en/news/releases/1.4.x/announcing-1.4.10/index.md:10]: This came from the announcement for 1.5.5, but I changed the version numbers.

Comment (ericvn) [reply content/en/news/releases/1.4.x/announcing-1.4.10/index.md:10]: OK. I noticed Francois had it as well. I'll ask in today's WG call about where this should be used.

**Pull request merged**